File: Matlab/04/1/Intro.m
s = tf('s')
h1 = 1/((1+2*s)*(1+3*s))
h2 = 1/(1+3*s)
h1 + h2
minreal(h1+h2)
H = minreal(h1/(1 + h1 * h2))
H = feedback(h1, h2)

File: Python/0.8/2/Intro.py
from control.matlab import ss
import numpy as np
a = np.matrix('''0 1 0 ;
-0.0071 -0.111 0.12;
0 0.07 -0.3''')
b = np.matrix(''' 0 ; -0.095; 0.072''')
c = np.matrix(''' 1.0 0 0;
0 1 0;
0 0 1;
1 0 -1''')
d = np.zeros((4, 1))
sys2 = ss(a, b, c, d)
k = np.matrix( '0  -0.67 0  0')
sysclosed = sys2.feedback(k)
import scipy.linalg as la
print (la.eig(sysclosed.A)[0])
print (la.eig(sys2.A)[0])